packager/mpd/base/representation.cc

```diff
@@ -297,6 +305,15 @@ void Representation::SetPresentationTimeOffset(
   media_info_.set_presentation_time_offset(pto);
 }
 
+void Representation::SetAvailabilityTimeOffset() {
+  // Adjust the frame duration to units of seconds to match target segment duration.
+  const float frame_duration_sec = (float)frame_duration_ / (float)media_info_.reference_time_scale();
+  const float ato = mpd_options_.mpd_params.target_segment_duration - frame_duration_sec;
```

Comment:
I thought elsewhere I had seen that ATO would be segment time minus chunk time.  Why subtract frame time instead?  Did I misunderstand?

Reply:
Your understanding is correct! However, the `frame_duration = sample_duration` as seen in [SetSampleDuration](https://github.com/google/shaka-packager/blob/master/packager/mpd/base/representation.cc#L196).  Each chunk contains a single media sample, so `sample_duration = chunk_duration`.  I added a clarifying comment to help out here, since the relationships are not clear.